plugins/flytekit-ray/flytekitplugins/ray/task.py

```diff
@@ -89,7 +92,9 @@ def get_custom(self, settings: SerializationSettings) -> Optional[Dict[str, Any]
         ray_job = RayJob(
             ray_cluster=RayCluster(
                 head_group_spec=(
-                    HeadGroupSpec(cfg.head_node_config.ray_start_params) if cfg.head_node_config else None
+                    HeadGroupSpec(cfg.head_node_config.ray_start_params, cfg.head_node_config.k8s_pod)
```

Comment:
IMO [this](https://github.com/flyteorg/flyte/blob/f20b8aa082820c56c0c721670b54148afa7d36a4/flyteidl/protos/flyteidl/plugins/ray.proto#L10) is not great
cc @EngHabu @pingsutw 
I would have loved us to model it more like a json so that modifying it would be faster without needing a protobuf change. 

But i see what you are doing now

Reply:
but i donot want to block. I see that we have backend ray plugin proto already.

Reply:
I appreciate it thank you.